modules/wheelos_common_msgs/0.1.2/presubmit.yml

```diff
@@ -0,0 +1,19 @@
+matrix:
+  platform:
+  - centos7
```

Comment:
You probably want to remove centos7 and debian10 since their gcc versions are too old. Also change `macos` to `macos_arm64`.

Reply:
OK, I finally deleted macos, it seems that the bazel version is too low

Reply:
OK, already add mac_arm deps bazel 7.x